tpl/bootstrap5.php

```diff
@@ -0,0 +1,486 @@
+							<div class="mb-3">
+								<label for="passworddecrypt"><svg width="16" height="16" fill="currentColor" aria-hidden="true"><use href="img/bootstrap-icons.svg#eye" /></svg> <?php echo I18n::_('Please enter the password for this paste:') ?></label>
+								<input id="passworddecrypt" type="password" class="form-control" placeholder="<?php echo I18n::_('Enter password') ?>" required="required">
+							</div>
+							<button type="submit" class="btn btn-success btn-block"><svg width="16" height="16" fill="currentColor" aria-hidden="true"><use href="img/bootstrap-icons.svg#power" /></svg> <?php echo I18n::_('Decrypt') ?></button>
+						</form>
+					</div>
+				</div>
+			</div>
+		</div>
+		<div id="loadconfirmmodal" tabindex="-1" class="modal fade" role="dialog" aria-hidden="true">
+			<div class="modal-dialog" role="document">
+				<div class="modal-content">
+					<div class="modal-header">
+						<h5 class="modal-title"><?php echo I18n::_('This secret message can only be displayed once. Would you like to see it now?') ?></h5>
+						<button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="<?php echo I18n::_('Close') ?>"></button>
+					</div>
+					<div class="modal-body text-center">
+						<button id="loadconfirm-open-now" type="button" class="btn btn-success" data-dismiss="modal"><svg width="16" height="16" fill="currentColor" aria-hidden="true"><use href="img/bootstrap-icons.svg#cloud-download" /></svg> <?php echo I18n::_('Yes, see it') ?></button>
+					</div>
+				</div>
+			</div>
+		</div>
+<?php
+if ($QRCODE) :
+?>
+		<div id="qrcodemodal" tabindex="-1" class="modal fade" role="dialog" aria-hidden="true">
+			<div class="modal-dialog" role="document">
+				<div class="modal-content">
+					<div class="modal-header">
+						<h5 class="modal-title"><?php echo I18n::_('QR code') ?></h5>
+						<button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="<?php echo I18n::_('Close') ?>"></button>
+					</div>
+					<div class="modal-body">
+						<div class="mx-auto" id="qrcode-display"></div>
+					</div>
+				</div>
+			</div>
+		</div>
+<?php
+endif;
+if ($EMAIL) :
+?>
+		<div id="emailconfirmmodal" tabindex="-1" class="modal fade" role="dialog" aria-hidden="true">
+			<div class="modal-dialog" role="document">
+				<div class="modal-content">
+					<div class="modal-header">
+						<h5 class="modal-title"><?php echo I18n::_('Recipient may become aware of your timezone, convert time to UTC?') ?></h5>
+						<button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="<?php echo I18n::_('Close') ?>"></button>
+					</div>
+					<div class="modal-body row">
+						<div class="col-xs-12 col-md-6">
+							<button id="emailconfirm-timezone-current" type="button" class="btn btn-danger"><svg width="16" height="16" fill="currentColor" aria-hidden="true"><use href="img/bootstrap-icons.svg#clock" /></svg> <?php echo I18n::_('Use Current Timezone') ?></button>
```

Comment:
Nothing prevents me or you, but these are not our resources. If this is a concern to you, maybe [raise this as an issue with them](https://github.com/twbs/icons/issues?q=is%3Aissue+aria+is%3Aclosed)? I rely on their doc and resources to contain a reasonable default for the accessibility and do not want to start having to patch these resources manually on our end, every time I upgrade their libraries, collections, etc.

Reply:
The thing is the did. They just say it is an implementation detail/leave it to the dev implementing. As said, they explicitly advise on how to do it:
> Using color to add meaning only provides a visual indication, which will not be conveyed to users of assistive technologies – such as screen readers. Ensure that information denoted by the color is either obvious from the content itself (e.g. the visible text), or is included through alternative means, such as additional text hidden with the .visually-hidden class.

There is not a lot Bootstrap itself can do here. Also the icons repo is the wrong choice for a bug report, because you can use icons in a good accessible way and in a non-accessible way… the thing is it depends on your use case.

They do [a lot, actually](https://github.com/twbs/bootstrap/issues?q=is%3Aissue+accessibility+is%3Aclosed+label%3Aaccessibility), but they cannot do anything. And I'd argue accessibility and properly implementing the libraries as per their docs _is_ our resource. It's like writing alt texts on social media, sure, it's not fun, but a thing one should pay attention to in web development. Just because things work for a sighted user does not mean they work for everybody.

That said, of course we can add this as a point for later (when we can remove old templates), just noting it is important to be done.